core/env/docker-compose.yml

```diff
@@ -22,5 +22,16 @@ services:
       POSTGRES_PASSWORD:
     volumes:
       - postgres:/var/lib/postgresql/data
+  minio:
+    image: minio/minio:RELEASE.2023-02-22T18-23-45Z.fips
+    ports:
+      - 127.0.0.1:${MINIO_PORT:-0}:9001
+    environment:
+      MINIO_ROOT_USER: cocktail_offerer
+      MINIO_ROOT_PASSWORD:
+    command: server --console-address ":9001" /data
+    volumes:
+      - minio_storage:/data
 volumes:
   postgres:
+  minio_storage:
```

Comment:
Volume for postgres data is named `postgres`. It's not named `postgres_storage`. So why be inconsistent with it?

Reply:
Fixed